components/motor/gpio/motor_encoder.go

```diff
@@ -174,6 +179,7 @@ type EncodedMotor struct {
 	// .01 would ramp very slowly, 1 would ramp instantaneously
 	rampRate    float64
 	maxPowerPct float64
+	flip        int64
```

Comment:
Is it worth adding in a comment explaining that this will be either +1 or -1, depending on `dirFlip` in the config? I can argue either way. Maybe it's fine as-is.

Reply:
Added!